### ISSUE TITLE
Doesn't compile with 1.38 nightly 

### ISSUE THREAD

**Issue by diegogangl** [dtolnay/mashup#20]
I'm trying to build [PyO3](https://github.com/PyO3/pyo3) which depends on mashup and it's panicking here:

```
error: proc-macro derive panicked
   --> /home/januz/.cargo/registry/src/github.com-1ecc6299db9ec823/mashup-0.1.9/src/lib.rs:153:1
    |
153 | / proc_macro_item_decl! {
154 | |     #[doc(hidden)]
155 | |     mashup_macro! => mashup_macro_impl
156 | | }
    | |_^
    |
    = help: message: assertion failed: `(left == right)`
              left: `Some("#[allow(unused,")`,
             right: `Some("#[allow(unused")`
    = note: this warning originates in a macro outside of the current crate (in Nightly builds, run with -Z external-macro-backtrace for more info)

error: aborting due to previous error

error: Could not compile `mashup`.
warning: build failed, waiting for other jobs to finish...
error: build failed
```

Version:
rustc 1.38.0-nightly (00ee1b47f 2019-08-11)

**Comment by dtolnay** [comment]
I believe this was fixed in proc-macro-hack 0.4.2. Could you try pulling in that newer version?

```
$ cargo update -p proc-macro-hack -p proc-macro-hack-impl
```

**Comment by diegogangl** [comment]
I didn't know about `cargo update`. I thought updating Py03 in the toml file would also auto-update dependencies, but it makes sense. That fixed it, Thanks!
